dashboard/app/views/home/index.html.haml

```diff
@@ -50,7 +50,10 @@
               %button.btn.btn-default{type: 'submit', style: 'margin-top:0'}= t('add_teacher_form.submit')
           - else
             = t('teacher.title', count: current_user.teachers.length, name: '')
-            = raw current_user.teachers.map{|t| t[:name] + ' ' + link_to(t('teacher.remove'), remove_follower_path(teacher_user_id: t[:id], student_user_id: current_user.id), confirm: t('teacher.confirm.remove'), method: :post)}.join(', ')
+            -if current_user.encrypted_password?
```

Comment:
could you put in a space between the `if/else` and `-`

`- if current_user.encrypted_password?`